src/Tvheadend.cpp

```diff
@@ -2470,6 +2470,24 @@ void CTvheadend::ParseRecordingAddOrUpdate ( htsmsg_t *msg, bool bAdd )
   if ((str = htsmsg_get_str(msg, "fanartImage")) != NULL)
     rec.SetFanartImage(GetImageURL(str));
 
+  /* Extra checks for HTSP v32+ */
+  if (m_conn->GetProtocol() >= 32) {
+    /* 
+      Due to changes in HTSP v32, try to use subtitle 
+      as the description if the description and summary
+      failed. Clear the subtitle afterwards  to avoid
```

Comment:
This comment is wrong as it refeneces "summary" which is not avail for recordings.

Reply:
Should this not be removed as well then? Thats why I referenced it.

https://github.com/kodi-pvr/pvr.hts/blob/dc35ade7b4afd0edd65c4170aab981218bad79d2/src/Tvheadend.cpp#L2460-L2461